packages/relay/src/lib/eth.ts

```diff
@@ -792,7 +792,13 @@ export class EthImpl implements Eth {
           gas = call.gas;
         }
       }
-      
+
+      // Gas limit for `eth_call` is 50_000_000, but the Hedera network limit is 15_000_000
```

Comment:
Assuming it's safe to cap this we can leave it here for now.
Please add a trace log.
Also add a test  to verify the submitted transaction has the gas capped.